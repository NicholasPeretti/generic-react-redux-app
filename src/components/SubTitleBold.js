import SubTitle from './SubTitle'
import styled from 'styled-components'

const SubTitleBold = styled(SubTitle)`
  font-weight: bold;
`

export default SubTitleBold
