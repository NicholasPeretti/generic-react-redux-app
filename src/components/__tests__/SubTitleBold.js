import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import SubTitleBold from '../SubTitleBold'

describe('SubTitleBold', () => {
  PresentationalTest(SubTitleBold)

  test('Should match snapshot', () => {
    const wrapper = shallow(<SubTitleBold>This is a SubTitleBold</SubTitleBold>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
