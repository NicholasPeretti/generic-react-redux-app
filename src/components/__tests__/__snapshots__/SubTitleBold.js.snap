// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitleBold Should match snapshot 1`] = `"<h3 class=\\"sc-bdVaJa sc-bwzfXH knfVVl\\">This is a SubTitleBold</h3>"`;
